src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -91,28 +93,31 @@ public int getTotalUserCount(){
   }
 
   /**
-   * Gets messages posted by a specific user.
+   * Gets messages received by the user. 
    *
-   * @return a list of messages posted by the user, or empty list if user has never posted a
-   *     message. List is sorted by time descending.
+   * @return a list of messages received by the recipient, or empty list if the recipient
+   * 	hasn't received message. List is sorted by time descending. 
    */
-  public List<Message> getMessages(String user) {
+  public List<Message> getMessages(String recipient) {
```

Comment:
So, this is changing getMessages() to return messages from the recipient, rather than the sender.  Is there any existing call site that is expecting the old behavior?  Is there a scenario (that you know of, that is plausible) in which you want from senders sometimes, and to recipients in others?  If so, then perhaps a refactoring could be helpful: getMessagesBySender and getMessagesForRecipient, both of which call a private getMessages which passes the query in as an argument.

I know you're filling in the stuff that as directed from the project pages, but don't hesitate to make this your own.

Reply:
I definitely noticed this when I was editing this method, and considered refactoring instead. Luckily, it seems that the direction they were taking in the project pages was that we would only want messages from recipients, but I think it is definitely better to refactor instead. Thanks!